app/controlplane/extensions/core/ociregistry/v1/README.md

```diff
@@ -0,0 +1,4 @@
+# OCI registry extension
+
+Use this extension to send attestations to a compatible OCI registry.
```

Comment:
only attestations? What about materials? Another ext?

Reply:
Maybe, as first version this sends attestations.